app/src/main/java/com/example/maintenancespace/models/cars/CarModel.java

```diff
@@ -14,6 +15,18 @@ public class CarModel {
     public CarModel(){
 
     }
+
+    public CarModel(String id, String vin, String make, String model, String trim, int year, String ownerId, ArrayList<String> userIds){
+        this.id = id;
+        this.vin = vin;
+        this.make = make;
+        this.model = model;
+        this.trim = trim;
+        this.year = year;
+        this.ownerId = ownerId;
+        this.userIds = userIds;
+    }
+
```

Comment:
Added a constructor that has the document id on it.  We should be using the document id to determine which car is which since the document id will be unique across the entire DB.